fabric-chaincode-shim/src/test/java/org/hyperledger/fabric/shim/fvt/ChaincodeFVTest.java

```diff
@@ -569,8 +570,10 @@ public Response invoke(final ChaincodeStub stub) {
 
         ChaincodeMockPeer.checkScenarioStepEnded(server, 3, 5000, TimeUnit.MILLISECONDS);
         assertThat(server.getLastMessageSend().getType(), is(TRANSACTION));
-        assertThat(server.getLastMessageRcvd().getType(), is(ERROR));
-        assertThat(server.getLastMessageRcvd().getPayload().toStringUtf8(), is("Wrong response2"));
+        assertThat(server.getLastMessageRcvd().getType(), is(COMPLETED));
+        String resp2 = ProposalResponsePackage.Response.parseFrom(server.getLastMessageRcvd().getPayload()).getMessage().toString();
+        System.out.println(resp2);
```

Comment:
Not a big issue, but did you intend to leave this here?

Reply:
no; I wasn't sure if the CI tests would work the same so just a test.